src/pages/valvonta-kaytto/toimenpide-osapuoli.svelte

```diff
@@ -23,8 +23,14 @@
   export let type;
 
   const types = {
-    henkilo: { pdf: valvontaApi.url.documentHenkilo },
-    yritys: { pdf: valvontaApi.url.documentYritys }
+    henkilo: {
+      pdf: valvontaApi.url.documentHenkilo,
+      courtAttachment: valvontaApi.url.courtAttachmentHenkilo
+    },
+    yritys: {
+      pdf: valvontaApi.url.documentYritys,
+      courtAttachment: valvontaApi.url.courtAttachmentYritys
+    }
```

Comment:
Tämä nyt on vain tällainen mietintä, mutta olisiko kivempaa, jos tässä pääsisi kutsumaan noita attachmenttejä näin

`courtAttachment: R.partial(valvontaApi.url.attachmentYritys, ["hallinto-oikeus.pdf"])`

Reply:
Mietin jotain vastaavaa itsekin aluksi, mutta ajattelin sitten mennä vain yksinkertaisemmalla toteutuksella, kun muiden liitteiden olemassaolo on toistaiseksi ihan teoreettinen mahdollisuus.